frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectPortletUtils.java

```diff
@@ -505,9 +504,8 @@ static AttachmentUsage mergeAttachmentUsages(AttachmentUsage u1, AttachmentUsage
     }
 
     public static String createVulnerabilityFriendlyUrl(ResourceRequest request) {
-        Portlet portlet = PortletLocalServiceUtil.getPortletById(PortalConstants.VULNERABILITIES_PORTLET_NAME);
-        Optional<Layout> layout = LayoutLocalServiceUtil.getLayouts(portlet.getCompanyId()).stream()
-                .filter(l -> ("/vulnerabilities").equals(l.getFriendlyURL())).findFirst();
+        Optional<Layout> layout = LayoutLocalServiceUtil.getLayouts(QueryUtil.ALL_POS, QueryUtil.ALL_POS).stream()
+                .filter(l -> ("/projects").equals(l.getFriendlyURL())).findFirst();
```

Comment:
Replace `/vulnerabilities` in place of `/projects`

Reply:
changed.